tensorlayer/prepro.py

```diff
@@ -1719,8 +1719,8 @@ def transform_matrix_offset_center(matrix, x, y):
     - See ``tl.prepro.rotation``, ``tl.prepro.shear``, ``tl.prepro.zoom``.
 
     """
-    o_x = float(x) / 2 + 0.5
-    o_y = float(y) / 2 + 0.5
+    o_x = float(x) / 2 - 0.5
```

Comment:
could you use 
```
o_x = (x - 1) / 2.0
o_y = (y - 1) / 2.0
```

to make it more readable?

Reply:
OK. I'll change it.